src/index.html

```diff
@@ -46,7 +46,7 @@
       <div class="collapse navbar-collapse" id="navbarNav">
         <ul class="navbar-nav ml-auto">
           <li class="nav-item">
-            <a class="nav-link" href="#calendar">Calendar</a>
+            <a class="nav-link" href="#past-clubs">Past Clubs</a>
```

Comment:
Can you change this to "Our Clubs" to match up with the header? Also change the id for consistency.